### PR TITLE
[CHANGE] outfit naming in the wiki

### PR DESCRIPTION
**Pull request by ClaudiaMia**
Relates to https://github.com/Project-Pandora-Game/pandora/pull/593

### CODE REVIEW

File: pandora-client-web/src/components/wiki/pages/items.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export function WikiItems(): ReactElement {
 				<li>Currently, everyone can put someone else into a room device if they are <a href='/wiki/characters/#CH_Character_permissions'>permitted</a> to.</li>
 				<li>Some room devices have <a href='#IT_Lock_module'>lock modules</a> that can for instance prevent a character from getting out of a room device slot.</li>
 				<li>You are unable to leave the room while your character occupies a character slot of a room device.</li>
-				<li>Room devices can also be stored in an <a href='#IT_Saving_outfits'>outfit template</a>, like regular items.</li>
+				<li>Room devices can also be stored in a <a href='#IT_Saving_collections'>collection template</a>, like regular items.</li>
```
Comment (Jomshir98) [inline pandora-client-web/src/components/wiki/pages/items.tsx:78]: Nitpick suggestion:
```suggestion
				<li>Room devices can also be stored in a <a href='#IT_Saving_collections'>saved items collection</a>, like regular items.</li>
```

File: pandora-client-web/src/components/wiki/pages/items.tsx [hunk 2]
```diff
@@ -139,27 +139,26 @@ export function WikiItems(): ReactElement {
...
+				well as the <a href='/wiki/spaces/#SP_Room_inventory'>room inventory</a> screen have an "Saved items"-tab that lets you access and manage all
+				your custom item collections stored on the Pandora server. A collection template can contain normal
+				items, <a href='#IT_Room-level_items'>room-level items</a>, body modifications, and even storage items.
 			</p>
 			<ul>
 				<li>
-					Creating an outfit: After you clicked the button to create a new outfit, you need to start moving the items that you want to be part of the outfit one-by-one
-					from the left side (your worn items or the room's inventory) to the right side (the temporary outfit template you just created).<br />
+					Creating a collection: After you clicked the button to create a new collection, you need to start moving the items that you want to be part of the collection one-by-one
+					from the left side (your worn items or the room's inventory) to the right side (the temporary collection template you just created).<br />
```
Comment (Jomshir98) [inline pandora-client-web/src/components/wiki/pages/items.tsx:152]: Nitpick:
```suggestion
					from the left side (your worn items or the room's inventory) to the right side (the temporary collection you just created).<br />
```

File: pandora-client-web/src/components/wiki/pages/items.tsx [hunk 3]
```diff
@@ -139,27 +139,26 @@ export function WikiItems(): ReactElement {
...
 				<li>
-					Storing an outfit: After you have completed your temporary outfit and given it a name, you can either save it on Pandora's server, if you have storage space left
+					Storing a collection: After you have completed your temporary collection and given it a name, you can either save it on Pandora's server, if you have storage space left
 					for your account, or you can export it in the form of a longer code, that can either be copied or downloaded, depending on what your device allows.
 				</li>
-				<li>Exported outfits can be stored on your device and later on be imported again or you can share the exported code with other users.</li>
+				<li>Exported collections can be stored on your device and later on be imported again or you can share the exported code with other users.</li>
 				<li>
-					Your outfit templates stored inside Pandora show a small live preview. You can increase the size of the previews in the "Interface"-tab of Pandora's settings page
+					Your collection templates stored inside Pandora show a small live preview. You can increase the size of the previews in the "Interface"-tab of Pandora's settings page
```
Comment (Jomshir98) [inline pandora-client-web/src/components/wiki/pages/items.tsx:161]: Same as higher:
```suggestion
					Collections stored inside Pandora show a small live preview. You can increase the size of the previews in the "Interface"-tab of Pandora's settings page
```

File: pandora-client-web/src/components/wiki/pages/items.tsx [hunk 4]
```diff
@@ -176,20 +175,20 @@ export function WikiItems(): ReactElement {
 				So the first item is usually something like a jacket or dress, whereas underwear is further down in the list.
 			</p>
 			<p>
-				You add and remove outfits worn by the character from top to bottom, so from the outermost worn item (e.g. a jacket or dress)
+				You add and remove the items worn by the character from top to bottom, so from the outermost worn item (e.g. a jacket or dress)
```
Comment (Jomshir98) [inline pandora-client-web/src/components/wiki/pages/items.tsx:178]: I'm actually not sure if "the" should be here:
```suggestion
				You add and remove items worn by the character from top to bottom, so from the outermost worn item (e.g. a jacket or dress)
```

File: pandora-client-web/src/components/wiki/pages/items.tsx [hunk 5]
```diff
@@ -176,20 +175,20 @@ export function WikiItems(): ReactElement {
...
 				towards the inner pieces, like how you would undress in reality, too.<br />
 				In case you dropped items in that manner to the <a href='/wiki/spaces/#SP_Room_inventory'>room inventory</a>, you can again add them to the body from the top to bottom,
 				so from the item worn the closest to the body, like you would start dressing in reality, too.
 			</p>
 			<p>
-				Now when you make an outfit template from something you wear, you need to start from the bottom, not from the top,
+				Now when you make a collection template from something you wear, you need to start from the bottom, not from the top,
```
Comment (Jomshir98) [inline pandora-client-web/src/components/wiki/pages/items.tsx:184]:
```suggestion
				Now when you make a saved item collection from something you wear, you need to start from the bottom, not from the top,
```

**Review by Sekkmer — APPROVED** [review]

File: pandora-client-web/src/components/wiki/pages/items.tsx [hunk 6]
```diff
@@ -176,20 +175,20 @@ export function WikiItems(): ReactElement {
...
 				because you are not undressing your character, but you are "dressing" a mannequin template doll by means of copying your outfit.
 				So you need to start from the item worn closest to the body, therefore bottom-up.<br />
-				When you want to use an outfit template to dress your character, you again need to start from the bottom, as you need to start with the item
+				When you want to use a collection template to dress your character, you again need to start from the bottom, as you need to start with the item
```
Comment (Jomshir98) [inline pandora-client-web/src/components/wiki/pages/items.tsx:187]:
```suggestion
				When you want to use a saved item collection to dress your character, you again need to start from the bottom, as you need to start with the item
```

File: pandora-client-web/src/components/wiki/pages/items.tsx [hunk 7]
```diff
@@ -176,20 +175,20 @@ export function WikiItems(): ReactElement {
...
 				worn closest to the body, like you would start dressing in reality, too.
 			</p>
 			<p>
-				In summary, the general dressing and undressing direction is top-down, but outfit template related directions are the opposite:<br />
+				In summary, the general dressing and undressing direction is top-down, but collection template related directions are the opposite:<br />
```
Comment (Jomshir98) [inline pandora-client-web/src/components/wiki/pages/items.tsx:191]:
```suggestion
				In summary, the general dressing and undressing direction is top-down, but saved item collection related directions are the opposite:<br />
```

**Review by Jomshir98 — COMMENTED** [review]
Saw a few possible improvements as the wording changed:

**Pull request merged**